tests/hikari/impl/test_entity_factory.py

```diff
@@ -25,6 +25,7 @@
 import mock
 import pytest
 
+from hikari import Locale
```

Comment:
```suggestion
from hikari import locale
```

And then access as such